unittests/test_utility.py

```diff
@@ -488,6 +490,16 @@ def test_import_from_file_load_unknown_path():
         assert '/foo' == e.path
 
 
+def test_import_from_file_load_unknown_path_unique_name():
+    try:
+        util.import_module_from_file('/foo', unique_name=True)
+        pytest.fail()
+    except ImportError as e:
+        module_hash = sha256('/foo'.encode('utf-8')).hexdigest()[:8]
+        assert f'rfm_foo_{module_hash}' == e.name
+        assert '/foo' == e.path
```

Comment:
The way the test name is constructed it should be irrelevant for the user. We need to test the behaviour of this mechanism. I would suggest the following:

1. Create a temporary Python file in `tmp_dir` and name it `os.py` and define a variable.
2. Try to load it with this function using its absolute path.
3. The test should load and return correctly our  `os.py` and not the already imported one (instead of `os.py` you could name it as any module that we have loaded already)

The same approach should be take with the a dummy valid test, so that we check that the loader loads it. Currently, this is not the case because the loader does not set `unique=True`, so the bug that this PR tries to solve is likely still there ;-)